protocols/mp-ecdsa/src/constants.rs

```diff
@@ -0,0 +1,64 @@
+// Constants for dkg-gadget
+
+// ================= Common ======================== //
+pub const DKG_KEYGEN_PROTOCOL_NAME: &str = "/webb-tools/dkg/keygen/1";
+
+pub const DKG_SIGNING_PROTOCOL_NAME: &str = "/webb-tools/dkg/signing/1";
+
+// ================= Worker ========================== //
+pub mod worker {
+    pub const ENGINE_ID: sp_runtime::ConsensusEngineId = *b"WDKG";
+
+    pub const STORAGE_SET_RETRY_NUM: usize = 5;
+
+    pub const MAX_SUBMISSION_DELAY: u32 = 3;
+
+    pub const MAX_KEYGEN_RETRIES: usize = 5;
+
+    /// How many blocks to keep the proposal hash in out local cache.
+    pub const PROPOSAL_HASH_LIFETIME: u32 = 10;
+}
+
+// ============= Signing Manager ======================= //
+
+pub mod signing_manager {
+    use dkg_primitives::types::SSID;
```

Comment:
Please move any `dkg-primitives` into this repo and create a package for it. Some name ideas
- `tangle-mpc-primitives`
- `tangle-dkg-primitives`
open to other suggestions.